modules/web/src/app/shared/components/side-nav-field/component.ts

```diff
@@ -12,30 +12,43 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {AfterViewChecked, ChangeDetectorRef, Component, Input} from '@angular/core';
+import {AfterViewChecked, ChangeDetectorRef, Component, Input, OnInit} from '@angular/core';
 import {Router} from '@angular/router';
-import {ProjectSidenavMainSection, View} from '@app/shared/entity/common';
+import {AdminPanelMainSections, ProjectSidenavMainSection, AdminPanelView, View} from '@app/shared/entity/common';
 import {shrinkGrow} from '@shared/animations/grow';
-
 @Component({
   selector: 'km-side-nav-field',
   templateUrl: './template.html',
   styleUrls: ['./style.scss'],
   animations: [shrinkGrow],
 })
-export class SideNavExpansionMenuComponent implements AfterViewChecked {
+export class SideNavExpansionMenuComponent implements AfterViewChecked, OnInit {
   private _expanded = false;
   readonly view = View;
-  @Input() label: ProjectSidenavMainSection;
+  @Input() label = '';
+  readonly projectSidenvMainSections = ProjectSidenavMainSection;
+  readonly adminPanelView = AdminPanelView;
+  readonly adminPanelMainSections = AdminPanelMainSections;
   @Input() icon = '';
   @Input() isSidenavCollapsed: boolean;
+  @Input() lastItem = false;
 
   constructor(private _router: Router, private readonly _cdr: ChangeDetectorRef) {}
 
   get expanded(): boolean {
     return this._expanded;
   }
 
+  get id(): string {
+    return this.label.replace(' ', '-');
+  }
+
+  ngOnInit(): void {
+    if (this.label === ProjectSidenavMainSection.Resources) {
+      this._expanded = true;
```

Comment:
```suggestion
      this._expanded = this.label === ProjectSidenavMainSection.Resources;
```
if statement can be removed by using this.